smsapi/smsapi.csproj

```diff
@@ -30,4 +30,7 @@
   <ItemGroup>
     <Folder Include="Api\Action\Contacts\" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="RestSharp" Version="106.13.0" />

```

Comment:
Please bump version, in the meanwhile new version has been resealed :)

Reply:
1e63cbf